packages/webapp/components/WebSeo.tsx

```diff
@@ -0,0 +1,11 @@
+import { getUnreadTitle } from '@dailydotdev/shared/src/components/notifications/utils';
+import { useNotificationContext } from '@dailydotdev/shared/src/contexts/NotificationsContext';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import React, { ReactElement } from 'react';
+
+function WebSeo({ title, ...props }: NextSeoProps): ReactElement {
+  const { unreadCount } = useNotificationContext();
+  return <NextSeo {...props} title={getUnreadTitle(unreadCount, title)} />;
```

Comment:
Did you check `titleTemplate` ? Maybe it would be cleaner?

Reply:
Not yet. Will check the link Ido gave then will make changes here. Thank you!

Reply:
Ahh, didnt see Ido mentioned the same. He beat me to it ^_^